### ISSUE TITLE
Support Spell check focus on data privacy for Flutter's text field

### ISSUE THREAD

**Issue by hoangdat** [linagora/tmail-flutter#3353]
### Desc:
- support spell check for Flutter text field without sending data to external server

### DoD:
- [ ] spell check: display error for error word
- [ ] spell check work locally, not depend on network

**Comment by chibenwa** [comment]
The time we get it, can we drop the feature ASAP?

**Comment by hoangdat** [comment]
yes, disabling is on going: #3352. This is for improvement later

**Comment by hoangdat** [comment]
- [ ] cherry pick to master branch also

**Comment by dab246** [comment]
> * [x]  cherry pick to master branch also

Implement at https://github.com/linagora/tmail-flutter/pull/3356
